core/src/main/java/nl/nn/adapterframework/pipes/IncreaseIntegerPipe.java

```diff
@@ -52,6 +59,20 @@ public PipeRunResult doPipe(Message message, IPipeLineSession session) throws Pi
 
 		String sessionKeyString = (String) session.get(sessionKey);
 		Integer sessionKeyInteger = Integer.valueOf(sessionKeyString);
+
+		ParameterValueList pvl = null;
+		if (getParameterList() != null) {
+			try {
+				pvl = getParameterList().getValues(message, session);
+			} catch (ParameterException e) {
+				throw new PipeRunException(this, getLogPrefix(session) + "exception extracting parameters", e);
+			}
+		}
+		ParameterValue pv = pvl.getParameterValue(INCREMENT);
+		if(pv != null && pv.getValue() != null) {
+			increment = pv.asIntegerValue(increment);
+		}
```

Comment:
The parameter value is meant to be used at runtime, so in doPipe(), and not in configure()

Reply:
You could check (during configure) if the setter has been set/changed and if a parameter has been supplied. If so the setter will be redundant.